### ISSUE TITLE
Multiple grafana organizations in one oncall installation

### ISSUE THREAD

**Issue by logicall** [grafana/oncall#598]
Hello!

Please tell me if this is possible (I did not manage to do it myself)
There is an installation of grafana and oncall. Grafana has 2 organizations. The first organization is integrated with oncall. How can I integrate the second organization there? The plugin is installed in grafana. If I enter a new invite code in the second organization, then the integration in the first is lost. Is it generally possible to integrate several organizations into one oncall installation?

Thank you!

**Comment by logicall** [comment]
I need this to sync teams and users. And they did not intersect in one oncall installation

**Comment by vadimkerr** [comment]
Hello 👋 
This is not possible at the moment, as stated in the Grafana OnCall OSS docs: https://grafana.com/docs/oncall/latest/open-source/#production-environment

> Note: The Grafana OnCall engine currently supports one instance of the Grafana OnCall plugin at a time.

**Comment by logicall** [comment]
Hello! Thanks for the answer.
Missed this when reading the documentation.
Then I have another question - is it enough for me to raise the engine and celery for another organization? Do I need to add another vhost to rabbitmq, and another base for redis?

Thank you!

**Comment by iskhakov** [comment]
We haven't yet tested that, but separating data with different rabbitmq vhosts, redis base, and databases should be enough.

**Comment by logicall** [comment]
Thank you!
I will try...

**Comment by logicall** [comment]
It turned out to be done.
I have one grafana container running, one rabbitmq (made 2 different vhosts for two organizations in grafana), one redis (made 2 databases for two organizations) and one mysql (it has 2 onCall databases for two organizations)
I also launched one engine container, celery, oncall_db_migration for organizations, where I registered different databases, vhost each for their organization.

All other oncall settings in grafana according to the instructions in the documentation.

If anyone is interested, I can check out the docker-compose file.
